mopidy/file/library.py

```diff
@@ -47,38 +44,38 @@ def browse(self, uri):
         result = []
         local_path = path.uri_to_path(uri)
 
-        if local_path == 'root':
+        if str(local_path) == 'root':
```

Comment:
Sorry, I missed this one before. This compares a native string that's possibly full of non-ascii with unicode and is a `UnicodeWarning` on Py2.